.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Start condition e.g. installed packages
+2. Command (line) executed
+3. Error encountered
+
+Please link or attach the packages or spec files involved.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Output**
+If applicable, add copy of the output on the command line or a screenshots to help explain your problem.
+
+**Environment**
+ - OS / Distribution: [e.g. Fedora 13]
```

Comment:
Not that it really matters, but I'd use a bit newer Fedora version as an example :smile: